src/conponents/Header/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import cn from 'classnames';
+import React, { FormEvent } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  handleAddTodo: (event: FormEvent<HTMLFormElement>) => void;
+  titleField: React.RefObject<HTMLInputElement>;
+  title: string;
+  setTitle: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
See the comment in the `App.tsx` - these should be the state of the Header component